### PR TITLE
Update localytics

### PR DESCRIPTION
**Pull request by miguelangel-dev**
Updated `localytics` to version 5.5. Update how it is working to keep the compatibility from iOS 9 to iOS 12 (using noninterrupting notifications provisionally to the Notification Center)

**NOTE** do not merge, we have to test it and update the privacy texts about GPS - it is necessary from April 2019

### CODE REVIEW

File: ScalaDays/ScalaDays/AppDelegate.swift [hunk 1]
```diff
@@ -90,73 +90,131 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+
+    private func setupThirdParties(application: UIApplication, launchOptions: [UIApplicationLaunchOptionsKey: Any]?) {
+        localyticsPushNotifications(application: application, launchOptions: launchOptions)
+        localytics(application: application, launchOptions: launchOptions)
+        googleAnalytics(application: application)
+        crashlytics(application: application)
+        twitter(application: application)
+    }
+
+    // MARK: third parties
```
Comment (juancazalla) [inline ScalaDays/ScalaDays/AppDelegate.swift]: This `MARK` seems duplicated

File: ScalaDays/ScalaDays/AppDelegate.swift [hunk 2]
```diff
@@ -90,73 +90,131 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+    private func localyticsPushNotifications(application: UIApplication, launchOptions: [UIApplicationLaunchOptionsKey: Any]?) {
+        if #available(iOS 12.0, *), objc_getClass("UNUserNotificationCenter") != nil {
```
Comment (juancazalla) [inline ScalaDays/ScalaDays/AppDelegate.swift]: Why do we need `objc_getClass("UNUserNotificationCenter") != nil`? 
Should not `#available(iOS 12.0, *)` be enough?

The only reason I imagine we would need this check is a bug on iOS/Xcode/Swift, the check for the availability should ensure that `UNUserNotificationCenter` is available.

File: ScalaDays/ScalaDays/AppDelegate.swift [hunk 3]
```diff
@@ -15,58 +15,58 @@
 */
 
 import UIKit
-import Crashlytics
 import SVProgressHUD
+
+import UserNotifications
 import Localytics
+import Crashlytics
 import TwitterKit
```
[the remark this answers is not in the document]
Comment (miguelangel-dev) [reply ScalaDays/ScalaDays/AppDelegate.swift]: hehehe, just I moved code (even some imports), and in the end, it finished in this state. I haven't any special intention to do it.

File: ScalaDays/ScalaDays/AppDelegate.swift (hunk 2 above)
Comment (miguelangel-dev) [reply ScalaDays/ScalaDays/AppDelegate.swift]: I followed the official documentation for `Localytics` (3. Register for notifications): https://docs.localytics.com/dev/ios.html#register-for-token-ios

In fact, I think it is not necessary it was how to check for older Swift version - but in this case, working with the OS version, I do not have any idea except your comment (possible bug).

File: ScalaDays/ScalaDays/AppDelegate.swift (hunk 1 above)
Comment (miguelangel-dev) [reply ScalaDays/ScalaDays/AppDelegate.swift]: thanks!

File: ScalaDays/ScalaDays/AppDelegate.swift (hunk 2 above)
Comment (juancazalla) [reply ScalaDays/ScalaDays/AppDelegate.swift]: 👍

**Pull request merged**